app/models/burndown.rb

```diff
@@ -107,8 +107,8 @@ def make_series(name, units, data)
 
   def determine_max
     @max = {
-      :points => @available_series.values.select{|s| s.unit == :points}.flatten.compact.max || 0.0,
-      :hours => @available_series.values.select{|s| s.unit == :hours}.flatten.compact.max || 0.0
+      :points => @available_series.values.select{|s| s.unit == :points}.flatten.compact.reject(&:nan?).max || 0.0,
```

Comment:
Line is too long. [114/100]<br>Use the new Ruby 1.9 hash syntax.<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.